common/persistence/elasticsearch/elasticsearchVisibility.go

```diff
@@ -77,11 +77,11 @@ var (
 )
 
 // NewElasticSearchVisibilityManager create a visibility manager connecting to ElasticSearch
-func NewElasticSearchVisibilityManager(esClient es.Client, index string, config *config.VisibilityConfig, logger bark.Logger) p.VisibilityManager {
+func NewElasticSearchVisibilityManager(esClient es.Client, index string, config *config.VisibilityConfig, logger log.Logger) p.VisibilityManager {
 	return &esVisibilityManager{
 		esClient: esClient,
 		index:    index,
-		logger:   logger.WithField(logging.TagWorkflowComponent, logging.TagValueESVisibilityManager),
+		logger:   logger.WithTags(tag.ComponentESVisibilityManager),
```

Comment:
No tag.Component?

Reply:
this is a predefined tagValue we made in our first refactoring PR.https://github.com/uber/cadence/commit/e1eae286d4305ec9f1ed70744b8c28267bbacb6b#diff-53c1076ea850f00548dda5f08f711613R93